test/src/unittests/rhythm/test_onsetdetection.py

```diff
@@ -128,12 +151,14 @@ def testEmpty(self):
         self.assertComputeFails(OnsetDetection(), spectrum, phase)
         spectrum = ones(1024)
         self.assertComputeFails(OnsetDetection(method='complex'), spectrum, phase)
-
-
+        self.assertComputeFails(OnsetDetection(method='complex_phase'), spectrum, phase)
+    
     def testDifferentSizes(self):
```

Comment:
Update the name of this test to represent that it is only for "complex*" tests. For example, `testComplexInputSizeMismatch` is more explanatory.